indium-chrome.el

```diff
@@ -66,20 +66,26 @@ Open URL if provided."
                                       nil
                                       'indium-chrome-url-history
                                       (car indium-chrome-url-history))))
-  (make-process :name "indium-chrome-process"
-                :command (list (indium-chrome--find-executable)
-                               (format "--remote-debugging-port=%s" indium-chrome-port)
-                               (or url "")))
+
+  (let ((executable (indium-chrome--find-executable)))
+    (if executable
+        (make-process :name "indium-chrome-process"
+                      :command (list executable
+                                     (format "--remote-debugging-port=%s" indium-chrome-port)
+                                     (or url "")))
+      (if (string-equal system-type "darwin")
+          (shell-command (concat "open -a Google\\ Chrome --args "
+                                 (format " --remote-debugging-port%s" indium-chrome-port)
```

Comment:
It's possible to perform a [command injection attack](https://www.owasp.org/index.php/Command_Injection) by setting `indium-chrome-port` to something like `9222 & rm ~/Downloads/*`

I'm not sure whether it matters, but it might be worth changing the call to `shell-command` to be a call to `make-process` instead.

Reply:
Also, is `" --remote-debugging-port%s"` missing an `=` between `port` and `%s`?

Reply:
Correct. I couldn't run with `open` from `make-process` (don't know why yet), but I can do a little more testing on this.  If it don't work with make-process, some validation would be required.

Yeah, probably removed by accident testing with Firefox which is `--start-debugger-server [port]`. :)

Reply:
Fixed the `--remote-debugging-port` part.

Reply:
It's probably not a big deal. Emacs packages generally aren't hardened against those kinds of attacks anyway.

By the way, feel free to stop by in chat if you'd like. https://gitter.im/indium-emacs/Lobby

Reply:
Thanks for reviewing it. I'll run more tests and if necessary I'll add a checker for the `indium-chrome-port `.